### DISCUSSION TITLE
Still bannable in 4.6?

### DISCUSSION THREAD

**Discussion by Overdawn4** [34736384/genshin-fps-unlock#390] [Q&A]
since there has just been an update to the game, might there have been changes to the anticheat and genshin detecting the program?

**Comment by Iwannabemercy** [comment]
It being detected isn't necessarily the issue. The official hoyolab or genshin statement for their "Fair Use" statement says that 
1) The use of plug-ins, accelerators, and any other third-party tools that affect the fairness of the game is strictly forbidden.

 COGNOSPHERE PTE. LTD. (herein referred to as "we") will respond to any such activity by taking measures including but not limited to removal of inappropriately obtained rewards, suspension of accounts, and permanent banning of accounts, with the precise measures to be decided based on the seriousness and number of violations.
 
 In short, this implies and states that specifically, the use of any plugins that affect the fairness of the game is a bannable offence. You getting some extra fps isn't something that does that, from what I understand. (Other than Ayato supposedly getting an extra slash, but that's nothing to worry about.)

**Comment by 34736384** [comment]
They have been aware of this tool since its release, so it is a no to your question. It's a different story if you inject other stuff in there
